Week2/homework2/queries.js

```diff
@@ -0,0 +1,87 @@
+const Query1 = `
+SELECT
+city.Name AS capital
+FROM
+country,
+city
+WHERE
+country.Capital = city.id
+AND country.Name = ? `;
+const Query2 = `
+SELECT DISTINCT
+countrylanguage.Language
+FROM
+country,
+countrylanguage
+WHERE
+countrylanguage.countrycode = country.code
+AND country.Region = ? `;
+
+const Query3 = `
+SELECT COUNT(*) as Number_Of_Cities 
+FROM city WHERE CountryCode 
+IN(SELECT CountryCode FROM countryLanguage WHERE Language = ?) `;
+
+const Query4 = `SELECT Name as Countries 
+FROM country WHERE region = ? and Code 
+IN(SELECT CountryCode FROM countryLanguage WHERE Language = ? and IsOfficial = 'T')`;
+
+const Query5 = `SELECT country.Continent,COUNT(distinct Language) as Number_Of_Spoken_Languages  
```

Comment:
This query is not working :)

Reply:
many thanks for your feedback.I will try to take care:)

Reply:
So you Query5 does not work??